lib/inc/internal/sources/cpuid_source.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <string>
+
+namespace whereami { namespace sources {
+
+    /**
+     * Register values returned by CPUID
+     */
+    struct cpuid_registers
+    {
+        /**
```

Comment:
I'd say these comments are redundant, but I think travis will fail if we don't have 100% docs coverage :/